### PR TITLE
feat: Threads

### PR DESCRIPTION
**Pull request by DjDeveloperr**
adds support for threads and api v9

### CODE REVIEW

**Comment by Helloyunho** [comment]
something happened but anyway 😉

File: mod.ts [hunk 1]
```diff
@@ -113,14 +113,15 @@ export {
   OverwriteType,
   OverrideType
 } from './src/types/channel.ts'
-export type {
-  OverwriteAsOptions,
-  OverwritePayload,
-  EmbedPayload
-} from './src/types/channel.ts'
+export type { EmbedPayload } from './src/types/channel.ts'
```
Comment (Helloyunho) [inline mod.ts:116]: why is it separated from other channel types

**Review by Helloyunho — CHANGES_REQUESTED** [review]

File: mod.ts (hunk 1 above)
Comment (DjDeveloperr) [reply mod.ts:116]: `export type`
image

Comment (Helloyunho) [reply mod.ts:116]: check the bottom

Comment (DjDeveloperr) [reply mod.ts]: fixed

**Review by Helloyunho — APPROVED** [review]
lgtm! i guess

**Pull request merged**